samples/modules/lvgl/accelerometer_chart/README.rst

```diff
@@ -33,6 +33,12 @@ Requirements
 Building and Running
 ********************
 
+First, ensure the optional LVGL module is enabled and available:
+
+   .. code-block:: console
```

Comment:
```suggestion
   .. code-block:: shell
```

I know this is kinda weird (and tons of occurrences will need to be updated in current docs eventually...) -- see the note here: https://docs.zephyrproject.org/latest/contribute/documentation/guidelines.html#code-and-command-examples about shell  vs. console.